### ISSUE TITLE
It seems wrong of Problem 5-1.b

### ISSUE THREAD

**Issue by run27017** [walkccc/CLRS#269]
The correct answer seems to be 99n, not 0.99n.

**Comment by yjian012** [comment]
I agree. "the variance of a binomial distribution" is when increment is 1, but if all values are multiplied by x, the variance will be multiplied by x^2, so it should be 100^2 np(1-p) which is 99n. It's easy to test it for small n like n=1.
